### DISCUSSION TITLE
Teensy 4.1

### DISCUSSION THREAD

**Discussion by Pacer27** [Rodemfr/MicronetToNMEA#83] [General]
Hello,

Has anybody a Setup with the Teensy 4.1 Board.
At the moment I am starting with this Projekt. It wold be fine to have a BoardConfig.h

Thank you

**Comment by dwarning** [comment]
Sorry for the delay. I have a 3.6 and 4.1 Teensy.
Attached is my BoardConfig.h usable for all 4 Teensy's.
I have a M8N but don't using the startup configuration feature. 
[BoardConfig_h.txt](https://github.com/Rodemfr/MicronetToNMEA/files/11436453/BoardConfig_h.txt)

**Reply by Pacer27** [reply]
Thank you.

**Comment by Pacer27** [comment]
Hello,
after a couple of days I got the system up and running. Thanks again.

A couple of my experiences:
Visual Code Version: 1.78.2 + PlatformIO Core 6.1.7-Home 3.4.3 did not work for me.
After the Teensy was programmed it always got stuck in a reboot loop.
There were also problems with the latest Arduino version.
 2.10.
Then I installed the Arduiono IDE 1.8.19 together with the Teensyduino version 1.57.

The Neo-8M I use (SPG3.51 PW 23.01) does not accept the autoconfiguration. The serial baudrate was always 9600. Only after I configured the module in U-center the data transfer worked.

I hope that these comments will help anyone who wants to start the project.

Thank you

**Reply by Rodemfr** [reply]
Thanks for the feedback,
We are observing a similar boot loop problem with latest PlatformIO/Teensyduino package (#84). It is unclear what happens but I suspect an illegal initialization code in MicronetToNMEA and a less tolerant version of TeensyDuino.
